boards.txt

```diff
@@ -11632,6 +11731,148 @@ heltec_wifi_lora_32_V2.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
 
+heltec_wifi_lora_V3.name=Heltec WiFi LoRa 32(V3) / Wireless shell(V3) / Wireless stick lite (V3)
+heltec_wifi_lora_V3.vid.0=0x303a
+heltec_wifi_lora_V3.pid.0=0x1001
+
+heltec_wifi_lora_V3.bootloader.tool=esptool_py
+heltec_wifi_lora_V3.bootloader.tool.default=esptool_py
+
+heltec_wifi_lora_V3.upload.tool=esptool_py
+heltec_wifi_lora_V3.upload.tool.default=esptool_py
+heltec_wifi_lora_V3.upload.tool.network=esp_ota
+
+heltec_wifi_lora_V3.upload.maximum_size=1310720
+heltec_wifi_lora_V3.upload.maximum_data_size=327680
+heltec_wifi_lora_V3.upload.flags=
+heltec_wifi_lora_V3.upload.extra_flags=
+heltec_wifi_lora_V3.upload.use_1200bps_touch=false
+heltec_wifi_lora_V3.upload.wait_for_upload_port=false
+
+heltec_wifi_lora_V3.serial.disableDTR=false
+heltec_wifi_lora_V3.serial.disableRTS=false
+
+heltec_wifi_lora_V3.build.tarch=xtensa
+heltec_wifi_lora_V3.build.bootloader_addr=0x0
+heltec_wifi_lora_V3.build.target=esp32s3
+heltec_wifi_lora_V3.build.mcu=esp32s3
+heltec_wifi_lora_V3.build.core=esp32
+heltec_wifi_lora_V3.build.variant=heltec_heltec_wifi_lora_V3
+heltec_wifi_lora_V3.build.board=heltec_heltec_wifi_lora_V3
```

Comment:
Change variant to `heltec_wifi_lora_32_V3` instead of `heltec_wifi_lora_V3` to match variant name folder and the board name the same, to match changes in previous comment.